### PR TITLE
Feature relationships

### PR DESCRIPTION
**Pull request by moelheni**
REF [18](https://trello.com/c/BGwKcQsi)

Now users can follow each others.

### CODE REVIEW

**Comment by moelheni** [comment]
@achr3f @SamiDghim  review this

File: app/helpers/relationship_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module RelationshipHelper
```
Comment (achr3f) [inline app/helpers/relationship_helper.rb:1]: remove this auto-generated file

File: db/schema.rb [hunk 2]
```diff
@@ -11,7 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
 ActiveRecord::Schema.define(version: 20160120134708) do
+=======
+ActiveRecord::Schema.define(version: 20160121084017) do
+>>>>>>> FEATURE Relationships (user follows others)
```
Comment (achr3f) [inline db/schema.rb]: Conflict !!

File: spec/factories/relationships.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :relationship do
+    follower_id 1
+followed_id 1
```
Comment (achr3f) [inline spec/factories/relationships.rb]: indentation

File: spec/helpers/relationship_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (achr3f) [inline spec/helpers/relationship_helper_spec.rb]: remove this file also

**Comment by achr3f** [comment]
OKI for me

File: app/controllers/relationship_controller.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  def followers
+    @followers = User.find( params[:user_id] ).followers
+  end
+
+  def following
+    @following = User.find( params[:user_id] ).following
+  end
+end
```
Comment (thameursaadi) [inline app/controllers/relationship_controller.rb:19]: user can also stop following other user

Comment (moelheni) [inline app/controllers/relationship_controller.rb:19]: :+1:  i will add this action

Comment (moelheni) [inline app/controllers/relationship_controller.rb:19]: done

File: app/helpers/relationship_helper.rb (hunk 1 above)
Comment (thameursaadi) [inline app/helpers/relationship_helper.rb:1]: try  --no-helper with generation of controller

**Pull request merged**